docker/Dockerfile.centos_centos7

```diff
@@ -0,0 +1,39 @@
+FROM centos:centos7
+
+RUN yum -y install epel-release
+
+RUN curl -O http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm
+RUN rpm -Uvh nux-dextop-release*rpm
+
+RUN yum -y install \
+        automake \
+        autoconf \
+        bison \
+        doxygen \
+        flex \
+        git \
+        gcc-c++ \
+        graphviz \
+        java-1.8.0-openjdk-devel \
+        libtool \
+        libsqlite3x-devel \
+        make \
+        mcpp \
+        ncurses-devel \
+        zlib-devel
+
+ARG GIT_USER
+ARG GIT_BRANCH
+ARG GIT_REVISION
+ARG SOUFFLE_OPTIONS
+
+WORKDIR /souffle
+RUN git clone https://github.com/${GIT_USER}/souffle /souffle
+RUN git checkout ${GIT_BRANCH}
+RUN git reset --hard ${GIT_REVISION}
```

Comment:
Is there an advantage to this instead of just `git checkout ${GIT_REVISION}`

Reply:
Nope, just my bad code - I will change it.